update_mkdocs_yml.py

```diff
@@ -44,6 +44,14 @@
     }
 
 mkdocs["extra"]["repo_name"] = mkdocs["repo_url"].replace("https://github.com/", "")
+mkdocs["extra"]["base_url"] = "https://docs.laminas.dev/"
+
+if mkdocs["extra"]["project"] == "Components":
+    mkdocs["extra"]["project_url"] = "https://docs.laminas.dev/components/"
+elif (mkdocs["extra"]["project"] == "MVC") or (mkdocs["extra"]["project"] == "Mvc"):
```

Comment:
_I think_ we are always using `Mvc`

Reply:
Unfortunately not, see: https://docs.laminas.dev/mvc/
It should be "MVC" but the initial build also introduced this problem.